src/VsixTesting/Remote.cs

```diff
@@ -49,6 +51,21 @@ public static void AutoKillWhenProcessExits(int processId)
             process.Exited += (_, e) => Process.GetCurrentProcess().Kill();
         }
 
+        public static void InitServiceProviderGlobalProvider()
+        {
+            ThreadHelper.Generic.Invoke(
+            () =>
+            {
+                var dte = ServiceProvider.GlobalProvider.GetService(typeof(SDTE)) as EnvDTE.DTE;
+                for (var shellVersion = new Version(dte.Version).Major; shellVersion >= 11; shellVersion--)
```

Comment:
❔  Why stop at 11? Seems at least 10 would be supported.

❔ Does version 13 (which never exists) throw an exception or silently fail? If it throws an exception, even if caught internally it could be annoying for debugging scenarios.

Reply:
> Why stop at 11? Seems at least 10 would be supported.

You're right! `VsixTesting` only supports down to 11 but someone might reference 10.

> Does version 13 (which never exists) throw an exception or silently fail?

The `Type.GetType(..., false)` should return false and it will be ignored. I didn't wan't it to fail if a particular shell version doesn't exist (even though they should all exist except 13).